modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -57,9 +57,29 @@ A standard {product-title} installation creates the following vCenter resources:
 ** 3 control plane nodes
 ** 3 compute machines
 
-Although these resources use 856 GB of storage, the bootstrap node is destroyed during the cluster installation process. A minimum of 800 GB of storage is required to use a standard cluster.
+These resources use 856 GB of storage in total.
 
-If you deploy more compute machines, the {product-title} cluster will use more storage.
+Bootstrap node resources are not configurable. The bootstrap node uses the following resources:
+
+* vCPU: 4
+* Virtual RAM: 16 GB
+* Storage: 120 GB
+
+The bootstrap node is destroyed before compute machines are created, so the maximum amount of resources used by cluster is the greater of the following:
+
+* {blank}
++
+----
+resources for control plane nodes + resources for bootstrap node
+----
+
+* {blank}
++
+----
+resources for control plane nodes + resources for compute machines
+----
```

Comment:
not sure if we really need this, but I am fine with leaving it in.

Reply:
I'll defer to Aditya on if this will help or not - if not no problem to remove it.